cl/corpus_importer/tasks.py

```diff
@@ -1508,6 +1508,7 @@ def make_attachment_pq_object(
     attachment_report: AttachmentPage,
     rd_pk: int,
     user_pk: int,
+    att_report_text: str = None,
```

Comment:
Shouldn't this be either `str | None` or `Optional[str]`?

Reply:
Yeah, I changed to str | None = None

Thanks!